### PR TITLE
added blog schema and keywords

### PR DESCRIPTION
**Pull request by anirudh-tantry**
added schema, publisher, keywords for each blog

### CODE REVIEW

File: pages/blog/_slug.vue [hunk 1]
```diff
@@ -144,6 +154,34 @@ export default {
       ],
     }
   },
+  jsonld() {
+    return {
+      '@context': 'https://schema.org',
+      '@type': 'Article',
```
Comment (ankitsinghaniyaz) [inline pages/blog/_slug.vue:160]: This should be blog post right?

File: pages/blog/_slug.vue [hunk 2]
```diff
@@ -144,6 +154,34 @@ export default {
...
+      mainEntityOfPage: {
+        '@type': 'WebPage',
+        '@id': `https://formester.com/blog/${this.article.slug}`,
```
Comment (ankitsinghaniyaz) [inline pages/blog/_slug.vue:163]:
```suggestion
        '@id': `https://formester.com/blog/${this.article.slug}/`,
```

should have a trailing slash as a convention on this site

File: pages/blog/_slug.vue [hunk 3]
```diff
@@ -144,6 +154,34 @@ export default {
...
+      author: {
+        '@type': 'Person',
+        name: this.article.author,
+      },
+      publisher: {
+        '@type': 'Organization',
+        name: 'Formester',
+        logo: {
+          '@type': 'ImageObject',
+          url: 'https://formester.com/_nuxt/img/logo.5f4f1b9.svg',
```
Comment (ankitsinghaniyaz) [inline pages/blog/_slug.vue:179]: How did you come to this url?

File: pages/blog/_slug.vue (hunk 1 above)
Comment (anirudh-tantry) [reply pages/blog/_slug.vue:160]: Oh sorry forgot to add I'll add now

File: pages/blog/_slug.vue (hunk 3 above)
Comment (ankitsinghaniyaz) [reply pages/blog/_slug.vue:179]: okay fine for now I guess but may have to update it as this is not a great way.

Comment (anirudh-tantry) [reply pages/blog/_slug.vue:179]: Updating now

**Review by ankitsinghaniyaz — APPROVED** [review]

**Pull request merged**